app/models/pocket.rb

```diff
@@ -16,8 +16,8 @@ class Pocket < ApplicationRecord
   end
 
   class << self
-    def unclassified
-      where(state: %w[Unweighed Weighed])
+    def unclassified_and_checked_in
```

Comment:
mmm i think you should add another scope query that includes the one you need (`unclassified`)